packages/contracts/test/voting/token-voting-setup.ts

```diff
@@ -1,7 +1,7 @@
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
 
-import {ERC20, ERC20VotingSetup} from '../../typechain';
+import {ERC20, TokenVotingSetup} from '../../typechain';
```

Comment:
ThIs file should have the tests for NFT for making it safer. We only use erc20Token address in this file

Reply:
This will be a separate task: https://discord.com/channels/672466989217873929/1045348071942410280/1045705111961358356